src/workerd/api/base64.c++

```diff
@@ -0,0 +1,45 @@
+#include "base64.h"
+
+#include "simdutf.h"
+
+#include <kj/debug.h>
+
+namespace workerd::api {
+
+kj::StringPtr simd_error_to_string(simdutf::error_code err) {
```

Comment:
This needs to return kj::String or you will see a use after free error in asan builds. Can you add a test that triggers this path?